src/components/ui/Heading/README.md

```diff
@@ -3,7 +3,7 @@ Headings are short pieces of text used to organise and structure a page's conten
 They provide both a semantic and a presentational role, guiding users (as well as search engines and other bots) through the page content both visually (through hierarchy) and structurally (through markup). They are especially important for users of assistive technology—heading allow quick navigation throughout a page.
 
 ```jsx
-<Heading>This just in: Scotland is rainy 🏴󠁧󠁢󠁳󠁣󠁴󠁿🌦</Heading>
+<Heading noMargin>This just in: Scotland is rainy 🏴󠁧󠁢󠁳󠁣󠁴󠁿🌦</Heading>
```

Comment:
Not a big fan of this, because it is the code that people see and can copy from our examples when clicking on the "Code" button below all of the examples.

It'd be nicer if the theme had these margins removed somehow, if that's possible. Or maybe even if the CSS for the component did it.

But this seems like it's not there to show off the component API; it's just there to fix a styling bug we have in the styleguide.